### PR TITLE
[#157422715] show cards with component

### PR DESCRIPTION
**Pull request by valentinacala**
the credit card component is properly implemented in the screens:
- ConfirmToProceedTransactionScreen 
- ChoosePaymentMethodScreen

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#157422715](https://www.pivotaltracker.com/story/show/157422715): Utilizzo di CreditCardComponent nella visualizzazione dei metodi di pagamento (carte di credito)

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: ts/api/wallet/wallet-api.ts [hunk 1]
```diff
@@ -250,6 +250,10 @@ export class WalletAPI {
     return transactions[id];
   }
 
+  public static getCard(index: number): Readonly<CreditCard> {
```
Comment (cloudify) [inline ts/api/wallet/wallet-api.ts]: there's exactly the same method in [this other PR](https://github.com/teamdigitale/italia-app/pull/242/files#diff-0dc7693a5ad51838135276b8279fbf10R273) - which PR is responsible for providing this method?

Comment (valentinacala) [reply ts/api/wallet/wallet-api.ts]: maybe I can just delete the method being replaced with selectors in next days

File: ts/screens/wallet/ConfirmToProceedTransactionScreen.tsx [hunk 2]
```diff
@@ -149,16 +178,36 @@ export class ConfirmToProceedTransactionScreen extends React.Component<
...
       </Container>
     );
   }
 }
+
+const mapStateToProps = (state: GlobalState): ReduxMappedStateProps => ({
+  card: selectedCreditCardSelector(state).getOrElse(UNKNOWN_CARD),
+  transaction: transactionForDetailsSelector(state).getOrElse(
+    UNKNOWN_TRANSACTION
+  )
```
[the remark this answers is not in the document]
Comment (valentinacala) [reply ts/screens/wallet/ConfirmToProceedTransactionScreen.tsx]: comment solved!

**Pull request merged**